server/Arcadia.Assistant.Health.Abstractions/HealthCheckType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Arcadia.Assistant.Health.Abstractions
+{
+    public enum HealthCheckType
+    {
+        Server,
+        Check1C,
```

Comment:
No 1C mentions outside CSPDb. I think the same for Database. Probably CSP implementation should return a dictionary<string, status> - 1c: ok, cspdb: ok

Reply:
Done.